### PR TITLE
Footer stick at the bottom

### PR DESCRIPTION
**Pull request by WOH2021**

### CODE REVIEW

File: app/views/offers/edit.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://images.unsplash.com/photo-1493225255756-d9584f8606e9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80);">
   <div class="container">
-    <h1>Change your trip:</strong></h1>
+    <h1>Edit yous trip:</strong></h1>
```
Comment (AristoRap) [inline app/views/offers/edit.html.erb:3]: just a typo here *yous*

File: app/views/offers/new.html.erb [hunk 2]
```diff
@@ -12,9 +12,8 @@
     <%= f.input :start_date %>
     <%= f.input :end_date %>
     <%= f.input :photos, as: :file, input_html: { multiple: true } %>
-    <%= f.button :submit,"Submit" class:"btn-smpl-form" %>
+    <%= f.button :submit, class:"btn-smpl-form" %>
+    <%= link_to "cancel", overview_path, :class=>"btn-ghost" %>
```
Comment (AristoRap) [inline app/views/offers/new.html.erb:16]: :class => ... is old syntax; best to write in the new way / class: '...'

File: app/views/offers/edit.html.erb [hunk 3]
```diff
@@ -13,8 +13,7 @@
     <%= f.input :end_date %>
     <%= f.input :photos, as: :file, input_html: { multiple: true } %>
     <%= f.button :submit, class:"btn-smpl-form" %>
+    <%= link_to "cancel", overview_path, :class=>"btn-ghost" %>
```
Comment (AristoRap) [inline app/views/offers/edit.html.erb:16]: :class => ... is old syntax; best to write in the new way / class: '...'

File: app/views/offers/new.html.erb (hunk 2 above)
Comment (WOH2021) [reply app/views/offers/new.html.erb:16]: done!

File: app/views/offers/edit.html.erb (hunk 3 above)
Comment (WOH2021) [reply app/views/offers/edit.html.erb:16]: done

File: app/views/offers/edit.html.erb (hunk 1 above)
Comment (WOH2021) [reply app/views/offers/edit.html.erb:3]: done

**Review by yauhenibankouski — APPROVED** [review]

**Pull request merged**